[docs] Migrate sorting (#233)
mui-org/material-ui-x: docs/src/pages/components/data-grid/pagination/pagination.md

## apiRef

> The Grid exposes a set of methods that will let you achieve all the above features using the imperative apiRef.

Reviewer: ```suggestion
The Grid exposes a set of methods that enables all of these features using the imperative apiRef.
```

> > ⚠️ Only use this API when you have no alternatives. Always start from the declarative API the Grid exposes.

Reviewer: ```suggestion
> ⚠️ Only use this API when you have no alternative. Always start from the declarative API that the Grid exposes.
```

- `setPageSize`
- `setPage`
- `onPageChange`
- `onPageSizeChange`

> Below is an example on how you can reset the page using the imperative `setPage` method.

Reviewer: ```suggestion
This example shows how you can reset the page using the imperative `setPage` method.
```

{{"demo": "pages/components/data-grid/pagination/ApiRefPaginationGrid.js"}}

> ## Paginate > 100 rows ✨

Reviewer: ```suggestion
## Paginate > 100 rows ✨
```

"✨" = "new feature". If this is meant to indicate a premium feature, it needs a more appropriate icon. (See suggestions elsewhere).

Author: mean to indicate premium, happy to use a different one, for now, the objective is to have at least one.

The `DataGrid` component can display up to 100 rows per page.

> The `XGrid` component removes this limitation, the following demo displays 200 rows per page:

Reviewer: ```suggestion
The `XGrid` component removes this limitation. The following demo displays 200 rows per page:
```
